src/helm/benchmark/compute_contamination_metrics.py

```diff
@@ -0,0 +1,212 @@
+import json
+import argparse
+from dataclasses import dataclass
+from typing import List
+from bitarray import bitarray
+from nltk import ngrams
+
+
+@dataclass(frozen=True, eq=False)
+class LightInstance:
+    """
+    A lighter `Instance` with only text fields.
+    """
+
+    input: str
+    """The input"""
+
+    references: List[str]
+    """References that help us evaluate"""
+
+
+@dataclass(frozen=True, eq=False)
+class LightScenario:
+    """
+    A lighter `Scenario`.
+    """
+
+    name: str
+    """The scenario name"""
+
+    light_instances: List[LightInstance]
+    """Instances of this scenario"""
+
+
+class BinaryScenarioMetric:
```

Comment:
I think this should be called `ContaminationStats` or something... (not metrics).  Not sure why we need `Binary` or `Scenario`.